packages/react-components/react-dialog/src/components/DialogContent/DialogContent.types.ts

```diff
@@ -0,0 +1,21 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+
+export type DialogContentSlots = {
+  root: Slot<'div', 'main'>;
+};
+
+/**
+ * DialogContent Props
+ */
+export type DialogContentProps = ComponentProps<DialogContentSlots> & {
+  /**
+   * Declares if focus will be trapped inside Dialog.
+   * @default modalType !== 'non-modal'
+   */
+  trapFocus?: boolean;
```

Comment:
Maybe don't expose this prop unless there's a need to ? The fact that the default value is `modalType !== 'non-modal' makes me think that the moday type is enough. We can always add it later if such an override is necessary